packages/e2e-test-utils/src/addTextElement.js

```diff
@@ -14,8 +14,12 @@
  * limitations under the License.
  */
 export default async function addTextElement() {
-  await expect(page).toClick('button[aria-label="Add new text element"]');
```

Comment:
@BrittanyIRL Tagging you here just to share that this PR will remove the text quick-add button and updates all the tests using it as well -- the button will soon not be available anymore for tests.

I'll be using the `Paragraph` instead when adding text since that's neutral styling-wise.

Reply:
Awesome, this also means that this line (which I neglected to make a proper TODO when I updated all the karma tests to check for accessibility issues) can run: https://github.com/GoogleForCreators/web-stories-wp/pull/9955/files#diff-7dc084a68522b10b39061c684b76c52b8c7e54e281adc120bbac8b5b4e891941R80